src/scripts/ci_check_install.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+"""This script is used to validate the results of `make install`"""
+
+import os
+import sys
+
+def has_extension(filename, extensions):
+    for ext in [ext for ext in extensions if ext != ""]:
+        if filename.endswith(".%s" % ext):
+            return True
+    return False
+
+def is_lib_file(filename):
+    return has_extension(filename, ["so", "a", "dll", "dylib", "lib"])
+
+def is_header_file(filename):
+    return has_extension(filename, ["h", "hpp", "h++", "hxx", "hh"])
+
+def main():
+    if len(sys.argv) < 2:
+        print("Usage: %s <install_prefix>" % sys.argv[0])
+        sys.exit(1)
+    install_prefix = sys.argv[1]
+
+    if not os.path.isdir(install_prefix):
+        print('Error: install_prefix "%s" is not a directory' % install_prefix)
+        sys.exit(1)
+
+    found_libs = False
+    found_headers = False
+
+    for (_, _, filenames) in os.walk(install_prefix):
```

Comment:
This logic seems easy to fool in the sense that any random directory with an `.h` and a `.so` will pass. A better test would be to have `build_config.json` export the list of installed headers, then the script could verify that every header so named is in the expected spot.

That said this at least verifies that something was installed at all and can always be improved from there.

Reply:
I suggest we improve the test script on master later and go with the simple version for now.

Reply:
Yeah sorry didn't mean to suggest you do that here, what you have is fine for now and we can do something a little nicer/more extensive later